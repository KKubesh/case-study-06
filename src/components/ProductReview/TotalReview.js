import React, {Component} from 'react';
import redStar from '../../images/red-star.png';

class TotalReview extends Component {

    render () {
        return (
            <div className="row-flex" data-testid="total-review">
                <div className="stars-overall" data-testid="star-overall">
                    <img src={redStar} alt="postive star 1"/><img src={redStar} alt="postive star 2"/><img src={redStar} alt="postive star 3"/><img src={redStar} alt="postive star 4"/><img src={redStar} alt="postive star 5"/> overall
                </div>
                <div className="total-reviews"  data-testid="total-reviews">
                    view all 14 reviews
                </div>
            </div>
        )
    }
}

export default TotalReview;