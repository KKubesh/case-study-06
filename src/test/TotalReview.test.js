import React from 'react';
import { render, cleanup } from 'react-testing-library';
import TotalReview from '../components/ProductReview/TotalReview';

afterEach(cleanup);

test('<TotalReview />', () => {
    const { getByTestId, container } = render(<TotalReview/>);
    expect(getByTestId('total-review')).toBeTruthy();
    expect(getByTestId('star-overall')).toBeTruthy();
    expect(getByTestId('total-reviews')).toBeTruthy();
    expect(container.firstChild).toMatchSnapshot();
});