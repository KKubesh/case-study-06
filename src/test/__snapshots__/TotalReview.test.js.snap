// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TotalReview /> 1`] = `
<div
  class="row-flex"
  data-testid="total-review"
>
  <div
    class="stars-overall"
    data-testid="star-overall"
  >
    <img
      alt="postive star 1"
      src="red-star.png"
    />
    <img
      alt="postive star 2"
      src="red-star.png"
    />
    <img
      alt="postive star 3"
      src="red-star.png"
    />
    <img
      alt="postive star 4"
      src="red-star.png"
    />
    <img
      alt="postive star 5"
      src="red-star.png"
    />
     overall
  </div>
  <div
    class="total-reviews"
    data-testid="total-reviews"
  >
    view all 14 reviews
  </div>
</div>
`;
